### PR TITLE
Add RTX 2060 Mobile specs

### PR DESCRIPTION
**Pull request by johko**
Adding a GPU poor option :wink: 
Specs with FP16 according to this page: https://www.techpowerup.com/gpu-specs/geforce-rtx-2060-mobile.c3348

### CODE REVIEW

**Review by julien-c — APPROVED** [review]

File: packages/tasks/src/hardware.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export interface HardwareSpec {
 	memory?: number[];
 }
 
-export const DEFAULT_MEMORY_OPTIONS = [8, 16, 24, 32, 40, 48, 64, 80, 96, 128];
+export const DEFAULT_MEMORY_OPTIONS = [6, 8, 16, 24, 32, 40, 48, 64, 80, 96, 128];
```
Comment (coyotte508) [inline packages/tasks/src/hardware.ts:34]: Should we change this line? It's not related IMO

Comment (pcuenca) [reply packages/tasks/src/hardware.ts:34]: I agree, that should not be necessary.

Comment (johko) [reply packages/tasks/src/hardware.ts]: yup, sorry somehow thought that was needed at first but you're totally right. reverted it

File: packages/tasks/src/hardware.ts [hunk 2]
```diff
@@ -112,6 +112,10 @@ export const SKUS = {
 				tflops: 16.6,
 				memory: [8],
 			},
+      "RTX 2060 Mobile": {
+				tflops: 9.22,
+				memory: [6],
+      }
```
Comment (coyotte508) [inline packages/tasks/src/hardware.ts:118]:
```suggestion
      },
```

**Pull request merged**